### PR TITLE
Schedule flat file write 2

### PR DESCRIPTION
**Pull request by mzappitello**
Create method to write a flat file from the database for a given service date. It joins the Vehicle Events, Vehicle Positions, and Static Stops tables into a format third parties can use to read performance data.

Add flat file writing to performance manager event loop after new data is generated from gtfs realtime files. Write flat files for service dates from newly processed data.

Asana Task: https://app.asana.com/0/1204931901750655/1204907226653447/f
Asana Task: https://app.asana.com/0/1204931901750655/1204907226653449/f

### CODE REVIEW

**Comment by mzappitello** [comment]
This Replaces https://github.com/mbta/lamp/pull/157

File: python_src/src/lamp_py/performance_manager/flat_file.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+)
+from lamp_py.postgres.postgres_utils import DatabaseManager
+
+
+def write_flat_files(
+    db_manager: DatabaseManager, dates: List[datetime.datetime]
+) -> None:
+    """write flat files to s3 for datetimes"""
+    dates_to_write = {get_service_date_from_timestamp(d) for d in dates}
+
```
Comment (rymarczy) [inline python_src/src/lamp_py/performance_manager/flat_file.py:27]: I think there is a chance that the service_date pulled from this function won't match the service_date's processed during the event loop. But I'm not totally sure on the edge cases that could cause issues here. 

Might be worth talking through the different variations of possible service_dates's returned compared to service_date's processed based on timezone and time of day variations.

**Review by rymarczy — CHANGES_REQUESTED** [review]
Looks pretty good, a few comments regarding the README documentation and a more general question regarding timestamps vs service_date for the writing of files in the event loop.

**Review by rymarczy — APPROVED** [review]
LGTM

**Pull request merged**